cincinnati/src/lib.rs

```diff
@@ -452,7 +452,10 @@ impl Graph {
 
     /// Get the edges expressed as version -> versions
```

Comment:
Lets update this, `include_abstract=true` would filter out abstract (== legacy releases not in cincinnati-graph-data) releases?

Reply:
Abstract releases are created by edges which refer to non-existing versions. I'd rather not discuss this here, but refer you to #188 and #256.

Reply:
(Changing the docstring is fine of course)